install error on pve 8.1 kernel version : 6.5.13-5

CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/i915_sriov_sysfs.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/gt/iov/intel_iov.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/gt/iov/intel_iov_debugfs.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/gt/iov/intel_iov_event.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/gt/iov/intel_iov_memirq.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/gt/iov/intel_iov_provisioning.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/gt/iov/intel_iov_query.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/gt/iov/intel_iov_relay.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/gt/iov/intel_iov_service.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/gt/iov/intel_iov_state.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/gt/iov/intel_iov_sysfs.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/hsw_ips.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_atomic.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_atomic_plane.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_audio.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_bios.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_bw.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_cdclk.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_color.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_combo_phy.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_connector.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_crtc.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_crtc_state_dump.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_cursor.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_display.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_display_power.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_display_power_map.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_display_power_well.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_dmc.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_dpio_phy.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_dpll.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_dpll_mgr.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_dpt.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_drrs.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_dsb.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_fb.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_fb_pin.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_fbc.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_fdi.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_fifo_underrun.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_frontbuffer.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_global_state.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_hdcp.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_hotplug.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_hti.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_lpe_audio.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_modeset_verify.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_modeset_setup.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_overlay.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_pch_display.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_pch_refclk.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_plane_initial.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_psr.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_quirks.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_sprite.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_tc.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_vblank.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_vga.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/i9xx_plane.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/skl_scaler.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/skl_universal_plane.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/skl_watermark.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_acpi.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_opregion.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_fbdev.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/dvo_ch7017.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/dvo_ch7xxx.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/dvo_ivch.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/dvo_ns2501.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/dvo_sil164.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/dvo_tfp410.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/g4x_dp.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/g4x_hdmi.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/icl_dsi.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_backlight.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_crt.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_cx0_phy.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_ddi.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_ddi_buf_trans.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_display_trace.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_dkl_phy.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_dp.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_dp_aux.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_dp_aux_backlight.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_dp_hdcp.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_dp_link_training.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_dp_mst.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_dsi.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_dsi_dcs_backlight.o
  CC [M]  /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_dsi_vbt.o
/var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_dp_mst.c: In function ‘intel_dp_mst_find_vcpi_slots_for_bpp’:
/var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_dp_mst.c:86:35: error: too many arguments to function ‘drm_dp_calc_pbn_mode’
   86 |                 crtc_state->pbn = drm_dp_calc_pbn_mode(adjusted_mode->crtc_clock,
      |                                   ^~~~~~~~~~~~~~~~~~~~
In file included from /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_display_types.h:36,
                 from /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_dp_mst.c:40:
./include/drm/display/drm_dp_mst_helper.h:835:5: note: declared here
  835 | int drm_dp_calc_pbn_mode(int clock, int bpp);
      |     ^~~~~~~~~~~~~~~~~~~~
/var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_dp_mst.c: In function ‘intel_dp_mst_mode_valid_ctx’:
/var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_dp_mst.c:906:17: error: too many arguments to function ‘drm_dp_calc_pbn_mode’
  906 |                 drm_dp_calc_pbn_mode(mode->clock, min_bpp, false) > port->full_pbn) {
      |                 ^~~~~~~~~~~~~~~~~~~~
./include/drm/display/drm_dp_mst_helper.h:835:5: note: declared here
  835 | int drm_dp_calc_pbn_mode(int clock, int bpp);
      |     ^~~~~~~~~~~~~~~~~~~~
make[2]: *** [scripts/Makefile.build:251: /var/lib/dkms/i915-sriov-dkms/6.5/build/drivers/gpu/drm/i915/display/intel_dp_mst.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [/usr/src/linux-headers-6.5.13-5-pve/Makefile:2039: /var/lib/dkms/i915-sriov-dkms/6.5/build] Error 2
make: *** [Makefile:234: __sub-make] Error 2
make: Leaving directory '/usr/src/linux-headers-6.5.13-5-pve'

--- a/drivers/gpu/drm/i915/display/intel_dp_mst.c
+++ b/drivers/gpu/drm/i915/display/intel_dp_mst.c
@@ -82,15 +82,8 @@ static int intel_dp_mst_find_vcpi_slots_for_bpp(struct intel_encoder *encoder,
 	}
 
 	for (bpp = max_bpp; bpp >= min_bpp; bpp -= step) {
-	  #if LINUX_VERSION_CODE < KERNEL_VERSION(6,6,14)
-		crtc_state->pbn = drm_dp_calc_pbn_mode(adjusted_mode->crtc_clock,
-						       dsc ? bpp << 4 : bpp,
-						       dsc);
-    #endif
-    #if LINUX_VERSION_CODE >= KERNEL_VERSION(6,6,14)
-    		crtc_state->pbn = drm_dp_calc_pbn_mode(adjusted_mode->crtc_clock,
-						       dsc ? bpp << 4 : bpp);
-    #endif
+	    crtc_state->pbn = drm_dp_calc_pbn_mode(adjusted_mode->crtc_clock,
+      						       dsc ? bpp << 4 : bpp);
 		drm_dbg_kms(&i915->drm, "Trying bpp %d\n", bpp);
 
 		slots = drm_dp_atomic_find_time_slots(state, &intel_dp->mst_mgr,
@@ -901,16 +894,10 @@ intel_dp_mst_mode_valid_ctx(struct drm_connector *connector,
 	ret = drm_modeset_lock(&mgr->base.lock, ctx);
 	if (ret)
 		return ret;
-	  #if LINUX_VERSION_CODE < KERNEL_VERSION(6,6,14)
-	    if (mode_rate > max_rate || mode->clock > max_dotclk ||
-	        drm_dp_calc_pbn_mode(mode->clock, min_bpp, false) > port->full_pbn) {
-    #endif
-    #if LINUX_VERSION_CODE >= KERNEL_VERSION(6,6,14)
-	    if (mode_rate > max_rate || mode->clock > max_dotclk ||
-	        drm_dp_calc_pbn_mode(mode->clock, min_bpp) > port->full_pbn) {
-    #endif
-		*status = MODE_CLOCK_HIGH;
-		return 0;
+	if (mode_rate > max_rate || mode->clock > max_dotclk ||
+	    drm_dp_calc_pbn_mode(mode->clock, min_bpp) > port->full_pbn) {
+	    *status = MODE_CLOCK_HIGH;
+	    return 0;
 	}
 
 	if (mode->clock < 10000) {

